lib/tasks/schools.rake

```diff
@@ -45,7 +45,7 @@ namespace :schools do
       download_button = download_form.button_with(value: "Results.zip")
       puts "'Results.zip' link found, downloading the file..."
       download_file = agent.click(download_button)
-      download_file.save("db/data/dfe-schools.zip")
```

Comment:
Is it worth putsing "Overwriting ..."?

Reply:
I've added a line about that to the task.